src/core.jl

```diff
@@ -86,6 +96,12 @@ function  fit!(chain, optimiser, loss, epochs,
     loss_func(x, y) = loss(chain(x), y)
     history = []
     prev_loss = Inf
+    if gpu
+        @info "Using GPU for training"
+        data = Flux.gpu.(data)
```

Comment:
Suggest suppressing the `@info` for `verbosity < 1`.

Reply:
Shouldn't this be `verbosity <  1 || @info "Using GPU for training` or `verbosity > 0 && @info "Using GPU for training"`.?

Reply:
fixed, my bad.